operators/src/source/ogr_source/mod.rs

```diff
@@ -535,110 +535,129 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
-    fn set_attribute_filters(&mut self, attribute_filters: &[AttributeFilter]) -> Result<()> {
+    fn create_attribute_filter_string(attribute_filters: &[AttributeFilter]) -> Option<String> {
+        Self::create_attribute_filter_string_int(
+            attribute_filters,
+            Self::create_range_filter_string,
+        )
+    }
+
+    fn create_attribute_filter_string_cast(
+        attribute_filters: &[AttributeFilter],
+    ) -> Option<String> {
+        Self::create_attribute_filter_string_int(
+            attribute_filters,
+            Self::create_range_filter_string_cast,
+        )
+    }
+
+    fn create_attribute_filter_string_int(
```

Comment:
why is the method called string_int?

Reply:
This was meant be be "string_internal". It takes an additional argument that maps the ranges. I renamed the method and added docs to clarify this: 49478117c52bc98231f78e96cdd7d32ea2bbc6bb